bchrpc/txfilter.go

```diff
@@ -207,6 +242,31 @@ func (f *txFilter) MatchAndUpdate(tx *bchutil.Tx, params *chaincfg.Params) bool
 
 	matched := f.matchAll
 
+	slpMsg, err := v1parser.ParseSLP(tx.MsgTx().TxOut[0].PkScript)
+	if err == nil {
```

Comment:
NIT: I find it odd to check err == nil... is slpMsg nil if the parse fails? Can we test if slpMsg != nil?

Reply:
fixed in 0aaf3ab, I also added a length check on `tx.MsgTx().TxOut`.

Reply:
I also have a bunch of local updates for the burn allowance, hope to push here soon.